tests/manage/pv_services/test_pvc_assign_node_pod.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def resources(request):
+    """
+    Deletes the pod resources created during the test
+
+    Returns:
+        list: empty list of pod
+    """
+    pods = []
+
+    def finalizer():
+        for pod in pods:
+            pod.delete()
+            pod.ocp.wait_for_delete(pod.name)
+
+    request.addfinalizer(finalizer)
+    return pods
+
+
+class TestNodeAssign(ManageTest):
+    """
+    Automates the following test cases:
+    OCS-717 - RBD: Assign nodeName to a POD
+    OCS-744 - CephFS: Assign nodeName to a POD
+    """
+
+    @tier1
+    @pytest.mark.parametrize(
+        argnames=["interface"],
+        argvalues=[
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL],
+                marks=pytest.mark.polarion_id("OCS-717")
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM],
+                marks=pytest.mark.polarion_id("OCS-744")
+            )
+        ]
+    )
+    def test_pvc_assign_node_pod(self, interface, pvc_factory, resources):
+        """
+        Test assign nodeName to a pod using RWO pvc
```

Comment:
Do you have test cases for RWX? if not, please consider

Reply:
Added CephFS RWX test.